src/pages/accounts/ForgotPassword.tsx

```diff
@@ -82,36 +129,160 @@ function ForgotPasswordPage() {
 	}, [email]);
 
 	const handleSubmit = async () => {
-		if (!validEmail(email)) {
-			setEmailErr(true);
+		// EMAIL
+		if (validEmail(email.value)) {
+			setEmail({ error: false });
+			setCodeStep(true); // Send validation code
+		} else {
+			email.errorMessage = EMAIL_ERROR_MESSAGE;
+			setEmail({ error: true });
+			return;
+		}
```

Comment:
Be careful here as this will send multiple codes when the use hits submit a second time for the code step. You can either split up the handleSubmit and have two submit versions, put the code step logic first so if the code step is set it will do that or assert that the codeStep is false when doing this